src/message/chain_message.rs

```diff
@@ -32,6 +32,10 @@ impl ChainMessage {
             ChainMessage::Signed(msg) => msg.cid(),
         }
     }
+
+    pub fn equal_call(&self, other: &Self) -> bool {
```

Comment:
Perhaps some docs? I have no idea what it does.

Reply:
Fixed